play27-json-ops/src/main/scala/play/api/libs/json/ops/JsResults.scala

```diff
@@ -0,0 +1,70 @@
+package play.api.libs.json.ops
+
+import play.api.libs.json.{JsError, JsPath, JsResult}
+
+import scala.collection.generic.CanBuildFrom
+import scala.language.higherKinds
+
+/**
+ * Useful helper methods for combining and filtering [[JsResult]]s.
+ */
+object JsResults {
```

Comment:
This was missing from the Play 2.7 / Scala 2.11 artifact, but is present in the Scala 2.13 one.

Reply:
Correction: This should already be available because of the dependency on play-json-common